src/expr/impl/src/udf/quickjs.rs

```diff
@@ -72,4 +85,22 @@ impl UdfImpl for QuickJsFunction {
             .call_table_function(&self.identifier, input, 1024)
             .map(|s| futures_util::stream::iter(s).boxed())
     }
+
+    fn create_state(&self) -> Result<ArrayRef> {
+        self.runtime.create_state(&self.identifier)
```

Comment:
```suggestion
    fn agg_create_state(&self) -> Result<ArrayRef> {
        self.runtime.call_agg_create_state(&self.identifier)
```

What about making the signatures more clear?